test/siphash_internal_test.c

```diff
@@ -40,7 +40,6 @@ typedef struct {
 
 static int benchmark_siphash(void)
 {
-# ifdef OPENSSL_CPUID_OBJ
```

Comment:
Since you removed the `OPENSSL_CPUID_OBJ` guard, you need to check the return value of `OPENSSL_rdtsc` instead (line 57).